### DISCUSSION TITLE
Old and Duplicate Texts

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#13870] [Bug reports]
### What happened?

There are several duplicate texts in the most recent update

- diving suit descriptions
- acid reaction name & description
- tonic liquid description
- cyanide & sufforin descriptions
- bandage, plastiseal & antibiotic glue descriptions
- saline description
- deusizine description
- broken diving suit name & description

and possibly more. I just checked most of the new stuff

### Reproduction steps

check any of the descriptions/names prior mentioned and observe that they switch between different lines

**Comment by Regalis11** [comment]
Thank you for the report! I created [a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13876), we'll look into it!

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, coming up in the next unstable build!
